app/src/main/java/org/hisp/dhis/android/dataentry/form/section/EditTextHintCache.java

```diff
@@ -0,0 +1,7 @@
+package org.hisp.dhis.android.dataentry.form.section;
+
+public interface EditTextHintCache {
```

Comment:
Idea is to keep the ViewHolder as simple as possible. Cache is its own class because it relies on Android Context so its not easily testable.